### ISSUE TITLE
Check on EALSTM prediction values

### ISSUE THREAD

**Issue by lindsayplatt** [USGS-R/lake-temp-lstm-static-data-release#51]
From Sam:

> - Seems like there are some unrealistic low temperatures – e.g., in the EALSTM w/NLDAS data predictions I frequently saw lakes with predicts <-5 deg C.  
> - Frequently saw a sharp decrease in temperature around day of year ~50 in the EALSTM predictions. Saw this in multiple lakes and multiple years, so doesn’t seem to be driven by one cold snap.

![Image](https://user-images.githubusercontent.com/13220910/225735244-5d96f251-06f2-4698-897d-69ecf1c718bc.png)

**Comment by AndyMcAliley** [comment]
I see these sudden dips in predicted temperature in most lakes. They don't occur every year. The cold temperatures usually last for only a day or two. They generally coincide with a quick drop in air temperature. I don't understand why those sudden dips in temperature are predicted, though.

I don't know of anything to do to "fix" the cold spikes at this point. We could, however, set all negative predicted temperatures to 0 C, if that makes physical sense.

**Comment by AndyMcAliley** [comment]
In that case, it sounds like it would be better to leave the predictions as they are. Here's a draft response:

"The EA-LSTM occasionally predicts sudden drops in lake temperatures which last for one or two days. While these sudden drops are physically improbable, we choose not to manually alter the neural network's predictions. That way, they represent a purely data-driven set of predictions that complement the physics-based GLM predictions. We have added comments on this behavior in the metadata."

And maybe we could add some text like this to the data release:

"The EA-LSTM was trained on temperature observations alone, without physics-based guidance. As a result, the model may produce some physically unrealistic predictions. We have observed sudden drops in lake temperature which last for one or two days in predictions. While these sudden drops are physically improbable, we choose to report the neural network's predictions unaltered. Therefore, the EA-LSTM predictions represent a purely data-driven set of predictions that complement the physics-based GLM predictions."
